modules/sievefilters/site.js

```diff
@@ -73,6 +73,30 @@ var hm_sieve_condition_fields = function() {
     };
 };
 
+var load_sieve_filters = function(pageName) {
+    const dataSources = hm_data_sources() ?? [];
+    if (dataSources.length) {
+        dataSources.forEach((source) => {
+            if(source.sieve){
+                let spinnerId = `spinner_${source.id}`;
+                let spinnnerText = hm_spinner_text(`${source.name}`, spinnerId);
+                $('#sieve_accounts').append(spinnnerText);
```

Comment:
Can you try to get one spinner while loading and remove it only when the last one gets loaded?

Reply:
From a UI perspective, this approach appears to be user-friendly, as it accounts for the fact that some items may take longer to process than others


https://github.com/user-attachments/assets/6e7e18e5-b87c-4fc8-abd6-b76c3318ba45